src/reducers.js

```diff
@@ -14,6 +19,7 @@ const createRootReducer = history =>
     authentication: identityReducer,
     configuration: identityReducer,
     userAccount,
+    [orderHistoryStoreName]: orderHistoryReducers,
```

Comment:
```
import {
  reducer as orderHistory
} from './order-history';

...
combineReducers({
  authentication: identityReducer,
  configuration: identityReducer,
  userAccount,
  orderHistory,
  router: connectRouter(history),
});
```

Might be a bit simpler. :)  I'm not so sure we need to worry about the actual key of the reducer being a variable yet... we don't really have any way of 'dynamically' adding pages or anything anyway.

Reply:
I think it's important for the "module" to know where to look for data in the redux store. If we `import reducer as orderHistory` then we're depending on the user of the module to know what the name should be for the selectors to work properly.

Reply:
Fair enough - I'd still make "orderHistoryReducers" singular.  It's one reducer being added, even if it delegates to other reducers.